gdx/src/com/badlogic/gdx/graphics/glutils/ETC1.java

```diff
@@ -114,10 +129,13 @@ public void write (FileHandle file) {
 					write.write(buffer, 0, bytesToWrite);
 					writtenBytes += bytesToWrite;
 				}
+
+				write.close();
+
+				gzip.close();
+
 			} catch (Exception e) {
-				throw new GdxRuntimeException("Couldn't write PKM file to '" + file + "'", e);
-			} finally {
-				StreamUtils.closeQuietly(write);

```

Comment:
Why would closing the file failed?
In Windows not closing file output stream locks the output folder until reboot or else.